perun-core/src/main/java/cz/metacentrum/perun/core/impl/AuthzResolverImpl.java

```diff
@@ -87,6 +91,43 @@ public class AuthzResolverImpl implements AuthzResolverImplApi {
 
 	private static final Pattern columnNamesPattern = Pattern.compile("^[_0-9a-zA-Z]+$");
 
+
+	private static final RowMapper<Map<String, Map<Integer, Group>>> AUTHZ_COMPLEMENTARY_OBJECTS_AUTH_GROUPS_MAPPER = (rs, i) -> {
```

Comment:
we should be able to represent the use case, when user is member of group A and group B, which are both voadmins of VO - then you need a list of groups to be related with one voadmin role. now only one of them is returned.